rusk-wallet/tests/mock.rs

```diff
@@ -0,0 +1,156 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+//! Mocks of the traits used in the wallet.
+
```

Comment:
I'm not going to review those since we're going to not include the mock in the final code and porting this code in the rusk's test.

Reply:
Take a gander at the implementation in the [repository](https://github.com/dusk-network/wallet-core).